tests/rptest/tests/full_disk_test.py

```diff
@@ -461,3 +461,77 @@ def test_basic_usage_report(self):
             self.redpanda.rolling_restart_nodes(self.redpanda.nodes)
 
             wait_until(lambda: self.check(0.05), timeout_sec=10, backoff_sec=5)
+
+
+class DiskStatsOverrideTest(RedpandaTest):
+    @cluster(num_nodes=3)
+    def test_disk_stat_total_overrides(self):
+        if not self.test_context.globals.get('use_xfs_partitions', False):
+            self.logger.warn(
+                "skipping test: looks like we are not using separate partitions for each node"
+            )
+            return
```

Comment:
Would it make sense to run these tests unconditionally, but with a cluster size of 1?

Reply:
I think it works ok now in all cases with the change I made. See comment on previous feedback.